lib/ansible/module_utils/common/arg_spec.py

```diff
@@ -130,30 +147,37 @@ def __init__(self, argument_spec,
                 self._valid_parameter_names.update([key])
 
     def validate(self, parameters, *args, **kwargs):
-        """Validate module parameters against argument spec. Returns a
-        ValidationResult object.
+        """Validate parameters against argument spec. Returns a :class:`ValidationResult` object.
 
-        Error messages in the ValidationResult may contain no_log values and should be
-        sanitized before logging or displaying.
+        Error messages in the :class:`ValidationResult` may contain no_log values and should be
+        sanitized with :func:`ansible.module_utils.common.parameters.sanitize_keys` before logging or displaying.
```

Comment:
maybe render it shorter?
```suggestion
        sanitized with :func:`~ansible.module_utils.common.parameters.sanitize_keys` before logging or displaying.
```